docs/pages/core-concepts.mdx

```diff
@@ -19,6 +19,31 @@ A minimal Teleport cluster consists of the **Teleport Auth Service** and
 services from a single `teleport` process on a [Linux
 host](linux-demo.mdx).
 
+## Teleport services
+
+A **Teleport service** manages access to resources in your infrastructure, such
+as Kubernetes clusters, Windows desktops, internal web applications, etc.
+
+A single running `teleport` process can run one or more **Teleport services**,
+depending on the user's configuration. Read about all subcommands of `teleport`
+in our [CLI Reference](./reference/cli/teleport.mdx). The **"Enabled by Default"** column indicates whether each service is active by default when you run `teleport start` without specifying services.
+
+## Teleport Agent services
```

Comment:
For some context, the reason this is an active work item right now is because during the recent CVE, people were unsure what the difference was between an agent and a client.

You can think of a Teleport deployment in terms of 3 things:
- The _control plane_. This refers to the auth_service and proxy_service. If you're a cloud customer, we (Teleport) run these for you.
- _Agents_ - Teleport processes running things like `app_service` or `windows_desktop_service`.
- _Clients_ - tools users use to connect to and interact with Teleport (Web UI, `tsh`, Teleport Connect, `event-handler`, etc)

When we say "agent" we never mean auth or proxy services (we refer to those as the control plane). For this reason, we should not include them in a section about agents.